chemprop/models/model.py

```diff
@@ -253,5 +266,20 @@ def load_from_file(cls, model_path, map_location=None, strict=True) -> MPNN:
 
         model = cls(**hparams)
         model.load_state_dict(state_dict, strict=strict)
+        
+        model.input_scalers = d["input_scalers"]
+        model.output_scaler = d["output_scaler"]
 
         return model
+
+
+class OutputTransform(object):
+
+    def __init__(self, output_scaler: StandardScaler | None=None):
+        self.output_scaler = output_scaler
+
+    def __call__(self, outputs):
+        if self.output_scaler is not None:
+            outputs = self.output_scaler.inverse_transform(outputs)
+
+        return outputs
```

Comment:
```suggestion
class OutputTransform(object):

    def __init__(self, output_scaler: StandardScaler):
        self.output_scaler = output_scaler

    def __call__(self, outputs):
        outputs = self.output_scaler.inverse_transform(outputs)

        return outputs
```
related suggestion about `test_step()`